.rubocop.yml

```diff
@@ -115,20 +119,14 @@ Metrics/ClassLength:
 Metrics/CyclomaticComplexity:
   Max: 10
 
-# Maximum line length of 120 chars
-Metrics/LineLength:
-  Max: 120
-
 # Avoid methods longer than 30 lines of code
 Metrics/MethodLength:
-  Exclude:
-    - 'engines/*/spec/**/*'
   Max: 30
 
 # Do not apply the module length check to spec files.
```

Comment:
nit: might want to update the comment as well here.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/419/commits/e2929f48e1c1a62380e2d66335e3ac2c0ec64403